rego/Chat.rego

```diff
@@ -241,39 +241,51 @@ if {
 #
 # Baseline GWS.CHAT.4.1v0.1
 #--
-GetFriendlyValue4_1(Value) := "OFF" if {
-    Value == "true"
-} else := "ON" if {
-    Value == "false"
-} else := Value
+default NoSuchEvent4_1(_) := true
+
+NoSuchEvent4_1(TopLevelOU) := true if {
+    Events := utils.FilterEvents(LogEvents, "RestrictChatProto restrictChatToOrganization", TopLevelOU)
+    count(Events) == 0
+}
+
+NoSuchEvent4_1(TopLevelOU) := true if {
+    # No such event...
```

Comment:
Recommend removing this comment for consistency (or else adding it to the rule on 246, but I don't think it adds much so my preference would be removing)
```suggestion
```